src/shared_modules/utils/windowsHelper.h

```diff
@@ -174,6 +192,53 @@ namespace Utils
         return ret;
     }
 
+    static std::string normalizeTimestamp(const std::string& timestamp1, const std::string& timestamp2)
+    {
+        std::string normalizedTimestamp;
+
+        if (timestamp1.size() == INSTALLDATE_REGISTRY_VALUE_SIZE)
+        {
+            if (isNumber(timestamp1))
+            {
+                size_t pos = timestamp2.find(" ");
```

Comment:
```suggestion
                size_t pos = timestamp2.find(' ');
```

Reply:
Changed, and also added the implementation of the stringHelper function you recommended here. 
https://github.com/wazuh/wazuh/pull/15000#discussion_r1015694321